### DISCUSSION TITLE
Kube vip version compatibility with kubernetes version

### DISCUSSION THREAD

**Discussion by rakeshk121** [kube-vip/kube-vip#995] [Q&A] [answered]
Hello ,

Im looking for the compatibility matrix for the kube-vip version with respect to kubernetes releases. Tried looking into docs, but could not find. Any pointers on this will be of help.

**Comment by thebsdbox** [comment, marked as answer]
At this time there isn't a compatibility matrix for kube-vip, however it's integrations with the API are very simplistic so it should work with anything from v1.0
